best practice 2/PdfBill.page

```diff
@@ -0,0 +1,166 @@
+<apex:page standardController="Invoice_Statement__c"
+    standardStylesheets="false" renderAs="pdf" sidebar="false"
+    showHeader="false" applyHtmlTag="false" applyBodyTag="false">
```

Comment:
What is the purpose of adding applyHtmlTag="false" applyBodyTag="false"? you cam aswer it here as a reply of this comment